.ci/common/test.sh

```diff
@@ -110,7 +110,16 @@ run_oldtests() {
 }
 
 run_single_includes_tests() {
-  ${MAKE_CMD} -C "${BULID_DIR}" check-single-includes
+  ulimit -c unlimited
+  if ! ${MAKE_CMD} -C "${BUILD_DIR}" check-single-includes; then
+    asan_check "${LOG_DIR}"
+    valgrind_check "${LOG_DIR}"
+    check_core_dumps
```

Comment:
`check-single-includes` did output everything: it is outputting what CMake builds and this is basically everything test actually does. `ulimit -c unlimited` making test pass is rather strange: this enables core dumps for `check_core_dumps` and it should be irrelevant because no core dumps are expected under any circumstances. Anybody has an idea what is going on here?